packages/demo-next/pages/blog/[slug].tsx

```diff
@@ -38,6 +38,13 @@ const addBlogPlugin: ContentCreatorPlugin<any> = {
     {
       name: 'title',
       component: 'text',
+      validate: value => {
```

Comment:
I'm not sure we want to commit these changes; looks like they are strictly the result of debugging.

Reply:
Down with Chris, either it works for the demo prupose or we don't ship it.

Reply:
Agreed, I'll prune the debugging changes from the next demo.

Reply:
@Enigmatical those changes have been pruned out.